test/intl_suggestors/constant_migrator_test.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:file/file.dart';
+import 'package:file/memory.dart';
+import 'package:over_react_codemod/src/intl_suggestors/intl_migrator.dart';
+import 'package:test/test.dart';
+
+import '../resolved_file_context.dart';
+import '../util.dart';
+
+void main() {
+  final resolvedContext = SharedAnalysisContext.overReact;
+
+  // Warm up analysis in a setUpAll so that if getting the resolved AST times out
+  // (which is more common for the WSD context), it fails here instead of failing the first test.
+  setUpAll(resolvedContext.warmUpAnalysis);
```

Comment:
"Today I learned"
This is a neat trick, where did you learn this?

Reply:
From the mui_migration_test.dart file that I copied it from. So probably Greg or Aaron?